test/e2e/performanceprofile/functests/README.txt

```diff
@@ -1,5 +1,16 @@
-HEADS UP!
+Functest consist in diffent suites:
 
-We have 2 test suites here, one running the "normal" performance tests, the other one performance profile update tests.
-The latter should run AFTER the former, and tests are executed in order of filenames.
-So be careful with renaming existing or adding new suites.
\ No newline at end of file
+0_config should be run before other in order to deploy PerformanceProfile and wait for MCP to pick MC and begin in updated state
+This deployment takes some time and requires reboot
+1- Performance performance test - should run after config and before update ones
+2- Performance update - Update test, should run after performance ones. 
+Different PerformanceProfiles and consequent updates to check if changes has been applied correclty
+3- Performance status - Check the status of differnet objects related with PAO
+4- Latency  // TODO
+5- Latency testing // TODO
+6- must-gather test - Check if must-gather cluster generated data is correct 
```

Comment:
we don't have a must-gather suite. am I missing something?

Reply:
Yep sorry the PR with 6- must-gather is waiting for revision. I will update this one after it